modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -509,10 +516,17 @@ public boolean isFormAvailable() throws PortalException {
 			if ((group != null) && group.isStagedRemotely()) {
 				ThemeDisplay themeDisplay = getThemeDisplay();
 
+				Role administratorRole = _roleLocalService.getRole(
```

Comment:
@rijema or @linolaoj, please rename the variable to `role` to keep consistency with existing code. Brian probably will ask it for us. :wink: